apps/nextjs-website/src/components/molecules/CkEditorPart/CkEditorPart.tsx

```diff
@@ -0,0 +1,17 @@
+'use client';
+import React from 'react';
+import { Box } from '@mui/material';
+
+export type CkEditorPartProps = {
+  content: string;
+};
+
+const CkEditorPart = ({ content }: CkEditorPartProps) => {
+  return (
+    <Box className={'container'} sx={{}}>
+      <div dangerouslySetInnerHTML={{ __html: content }}></div>
```

Comment:
Considering that the `content` comes from the backend, you should "sanitize" it.
Take a look at [DOMPurify](https://github.com/cure53/DOMPurify) and [Isomorphic DOMPurify](https://github.com/kkomelin/isomorphic-dompurify)